### PR TITLE
 #180884980 Create Top Navigation using existing components

### PR DESCRIPTION
**Pull request by RACHID-WEBDEV**
#### What does this PR do?
 - Create a Top Navigation using existing components
#### Description of Task to be completed?
- Create Navbar components
- Add a Logo SVG image
- import the button component
#### How should this be manually tested?
- Clone the repo
- Run `npm install` or `yarn install` in your terminal to install the required dependencies.
- Run `npm run dev` or  `yarn dev` to start 
- Open your browser and visit `http://localhost:3000/`

#### Screenshots

header

### CODE REVIEW

File: components/Navbar.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+const Navbar = () => {
+  return (
+    <header>
+      <nav className="flex items-center bg-[#F5F6F8]  h-[4.8125rem] p-8">
+        <div className="ml-16">
+          <Image src={Logo} alt="logo" />
+        </div>
+        <ul className="hidden sm:flex flex-1 justify-end items-center gap-1 mr-20">
+          <Button outline>Sign in</Button>
+          <Button>try it now</Button>
```
Comment (andela-hpopoola) [inline components/Navbar.js:15]: This should be written in the right case.

```suggestion
          <Button>Try it now</Button>
```

File: pages/index.js [hunk 2]
```diff
@@ -1,8 +1,12 @@
 import Button from '@/components/form/Button';
+import Navbar from '@/components/Navbar';
 
 export default function Home() {
   return (
     <>
+      <Navbar />
+      <br />
```
Comment (andela-hpopoola) [inline pages/index.js:8]: We should use a padding or margin bottom here instead of `<br />`

**Review by andela-hpopoola — CHANGES_REQUESTED** [review]
Thank you for working on this. 

I noticed a couple on this on best practices:
1. The branch name `ft-Create-Top-Navigation-using-existing-components-#180884980` can be improved to meet practises as follows: 
- your branch names should always be in lower case and not mixed case 
- The hash symbol before the Pivotal Tracker Id is not necessary
- - 2 or 3 can be used in this instance for the branch name (highest 5)

Something like `ft-create-top-navigation-180884980` should suffix for the branch name. Please note that no action is required from you on this, I am just pointing it out to ensure that you follow the conventions we agreed on here
 
https://github.com/andela/bestpractices/wiki/Git-naming-conventions-and-best-practices

2. It might also be better to include a screenshot the Navbar you have created in the PR description

3. Since the Storybook story has been approved, I think it is better to integrate this into Storybook too. You can read this Git workflow for more information on rebasing:

https://github.com/andela/bestpractices/wiki/Git-Workflow

Thank you for the hard work 💯 .

**Comment by andela-hpopoola** [comment]
Thank you for your hard work on this. A gentle reminder that you have awaiting comments on this PR. Feel free to work on it when you are chanced, I only wanted to be sure that I am not the one delaying your work on this. Thank you.

**Comment by andela-hpopoola** [comment]
Thank you for helping out with this. I have updated the branch here to be in sync with the current `main` branch to make it easier to work it for you. Thank you.

**Comment by RACHID-WEBDEV** [comment]
### Top Navigation component
header

**Comment by andela-hpopoola** [comment]
> ### Top Navigation component
> header


Awesome. Thank you for adding this but I think this should be added to the PR (I have added it above) 🙌🏽

**Review by andela-hpopoola — APPROVED** [review]
Thank you for the hard work on this. I believe it is safe to merge this now.

**Pull request merged**